src/controllers/agent.js

```diff
@@ -0,0 +1,66 @@
+import dotenv from 'dotenv';
+import Model from '../../../Desktop/property-pro-api/src/models/model';
+
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+
+const agentModel = new Model('agents');
+dotenv.config();
+
+export const Agentsignup = async (req, res) => {
+  const {
+    firstName, lastName, password, email, phoneNumber
+  } = req.body;
+
+  const columns = 'first_name, last_name, password, email, phone_number ';
+  const strong = await bcrypt.hash(password, 12);
+  const values = `'${firstName}', '${lastName}', '${strong}', '${email}', '${phoneNumber}'`;
+  const checkEmail = await agentModel.select('*', ` where email = '${email}' `);
```

Comment:
I can't see a validation for "Account already exist" Please validate that the user email has not been used by other users and throw the appropriate error message.